addons/web/controllers/home.py

```diff
@@ -61,6 +61,7 @@ def web_client(self, s_action=None, **kw):
         # Restore the user on the environment, it was lost due to auth="none"
         request.update_env(user=request.session.uid)
         try:
+            request.env["res.users"]._on_webclient_bootstrap()
```

Comment:
inside `web` we care about the quotes enough to bother you about them :D
```suggestion
            request.env['res.users']._on_webclient_bootstrap()
```

Reply:
But Julien, sthap !

Reply:
We are using `"` in discuss/mail and will also bother you. I guess it's fair.